integrations/qdrant/src/haystack_integrations/components/retrievers/qdrant/retriever.py

```diff
@@ -48,6 +49,7 @@ def __init__(
         :param top_k: The maximum number of documents to retrieve.
         :param scale_score: Whether to scale the scores of the retrieved documents or not.
         :param return_embedding: Whether to return the embedding of the retrieved Documents.
+        :param score_threshold: A minimal score threshold for the result.
```

Comment:
I would add a longer explanation. In particular, I would mention that
>  Score of the returned result might be higher or smaller than the threshold depending on the Distance function used. E.g. for cosine similarity only higher scores will be returned.

See https://python-client.qdrant.tech/qdrant_client.qdrant_client

Reply:
Will use the suggestion.